sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -748,6 +764,14 @@ object DateTimeUtils {
   }
 
   /**
+   * Add timestamp and days interval.
+   * Returns a timestamp value, expressed in microseconds since 1.1.1970 00:00:00.
+   */
+  def timestampAddDays(start: SQLTimestamp, days: Int): SQLTimestamp = {
+    timestampAddInterval(start, 0, days * MICROS_PER_DAY)
```

Comment:
is it just `start + days * MICROS_PER_DAY`? `timestampAddInterval` has some complex logic to handle month, which is not unnecessary here.